mobile/src/libs/packages/services/notification-service.ts

```diff
@@ -0,0 +1,48 @@
+import { type ValueOf } from 'react-native-gesture-handler/lib/typescript/typeUtils';
```

Comment:
should be in libs/packages/notification.ts without services folder, also rename the class to Notification